bigbluebutton-html5/imports/ui/components/toast/component.jsx

```diff
@@ -23,38 +23,91 @@ const defaultIcons = {
   [toast.TYPE.DEFAULT]: 'about',
 };
 
+const getContentStrings = (msgContent) => {
+  const MIN_LENGTH = 2;
+  const strings = [];
+  if (typeof msgContent === 'string') strings.push(msgContent);
+  if (typeof msgContent === 'object') {
+    const { children } = msgContent.props;
+    if (Object.keys(children).length === MIN_LENGTH) {
+      return getContentStrings(children[1]);
+    }
+    if (Object.keys(children).length > MIN_LENGTH) {
+      Object.entries(children).reduce((acc, [key]) => {
+        if (typeof children[key] === 'string') {
+          strings.push(children[key]);
+        }
+        return null;
+      });
+    }
+  }
+  return strings;
+};
+
+const formatMessageStrings = (messages) => {
+  const _messages = messages;
+  const MAX_MESSAGE_CHARS = 40;
+  _messages.forEach((item, index) => {
+    if (_messages[index].length > MAX_MESSAGE_CHARS) {
```

Comment:
Isn't `_messages[index]` the same as `item` ?

Reply:
The whole forEach() should probably just be a map() and don't use the index at all.